samples/snippets/create_http_task_test.py

```diff
@@ -46,3 +46,15 @@ def test_create_http_task(test_queue):
         TEST_PROJECT_ID, TEST_QUEUE_NAME, TEST_LOCATION, url
     )
     assert TEST_QUEUE_NAME in result.name
+
+    result = create_http_task.create_http_task(
```

Comment:
This is fine, but for named parameters I usually put the name in the function call as well as the definition. E.g., in_seconds=180 instead of just 180. But that's a personal preference, and this LGTM.